orca-core/src/main/java/com/netflix/spinnaker/orca/TaskExecutionInterceptor.java

```diff
@@ -50,4 +50,7 @@ default StageExecution beforeTaskExecution(Task task, StageExecution stage) {
   default TaskResult afterTaskExecution(Task task, StageExecution stage, TaskResult taskResult) {
     return taskResult;
   }
+
+  default void finallyAfterTaskExecution(
```

Comment:
can you add a doc comment here on the purpose of this hook point for implementers

Reply:
- [X] add docs